news/2023/2023-08-29-changes-to-the-mappers-guild.md

```diff
@@ -0,0 +1,149 @@
+
+## Ranked maps requirement lifted
+
+Previously, you needed **at least 3 Ranked maps** to host beatmaps and earn rewards in the Mappers' Guild. Those days are over.
+
+As of this change, **anyone can participate in the Mappers' Guild**!
+
+This includes:
+
+- Linking your Featured Artist beatmaps to the [Mappers' Guild map list](https://mappersguild.com/beatmaps) to track reward progress
+- Collaborating in [quests](https://mappersguild.com/quests)
+- Submitting maps to new [priority quests](https://mappersguild.com/missions)
+
+If you're a newer mapper looking to get started with the Mappers' Guild, feel free to ask questions in the [osu! Discord server](https://discord.com/invite/ppy)'s `#mappers-guild` channel!
+
+## Smaller changes
+
+Most changes to the Mappers' Guild aren't too glamorous. Here's a few highlights from the past year:
+
+### Beatmap Nominator modding points
+
+Up until now, modders would earn 1 [Mappers' Guild point](/wiki/Community/Mappers_Guild#rewards) for each map they mod.
+
+Some [Beatmap Nominators](/wiki/People/Beatmap_Nominators) have requested their points for modding to scale according to beatmap length, since anyone who nominates a beatmap is guaranteed to have modded the full mapset.
+
+That said, anyone who nominated a Mappers' Guild beatmap has earned points relative to their modding efforts. This includes cases when a user forgot to add themselves as a modder too!
+
+### Notifications and invites removed
+
+Anyone who engaged with the Mappers' Guild before could tell you that these were annoying. So now they're gone.
+
+### Automatic quest dropping
+
+It's no secret that a lot of [quests](https://mappersguild.com/quests) are inactive. There's no major penalty for this, but to give other users a chance, any quest that is inactive for a year is dropped and made available to the public.
+
+### Party leader can compensate points for other users
+
+To claim certain [quests](https://mappersguild.com/quests), each party member would need to spend some points. Some users have 0 available points though, so they can't take quests.
+
+Instead of completely locking users out of quests, the party leader can now compensate for another user's lack of points by spending more of their own points.
+
+## Major contributors
+
+A ton of people are involved in the **Mappers' Guild**, but some stand out above the crowd. Below are some of the top contributors to the guild! (More specifically, this is anyone who's contributed 10+ difficulties to Ranked Featured Artist maps since last July... and [SMOKELIND](https://osu.ppy.sh/users/9327302), who focused on total Ranked maps instead.)
+
+There's absolutely no way we could highlight every beatmap/quest, but we *can* shout out some of the **Mappers' Guild**'s top contributors. Each of these users contributed 5 or more ranked beatmaps since November 2020: <!-- this will need to be updated again later -->
```

Comment:
```suggestion
There's absolutely no way we could highlight every beatmap/quest, but we *can* shout out some of the **Mappers' Guild**'s top contributors. Each of these users contributed 5 or more Ranked beatmaps since November 2020: <!-- this will need to be updated again later -->
```